### ISSUE TITLE
DateTimePickerPlugin: A date/time picker dialog is already showing.

### ISSUE THREAD

**Issue by aesculus** [skwasjer/skwas-cordova-plugin-datetimepicker#48]
I am loading my date time picture as such:
`$("#plan_actual_a_time").siblings('a').on( "click", function(event) {
					var options = {mode: "time", 
						success: function(newDate) {
							$('#plan_actual_a_time').val(moment(newDate).format("HH:mm"));
							retain_arrival_time = newDate;
						}
					};
					if (retain_arrival_time) {
						options.date = retain_arrival_time;
					}
					cordova.plugins.DateTimePicker.show(options);
				});`

If I click my button then the DT picker shows. I can either dismiss it with the Done or Cancel.

If I click the button again I get the warning "DateTimePickerPlugin: A date/time picker dialog is already showing." in the console log.

What am I doing wrong here?

**Comment by skwasjer** [comment]
Hi, please add more context [as described in here](https://github.com/skwasjer/skwas-cordova-plugin-datetimepicker/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) to your OP.

**Comment by aesculus** [comment]
Appears to be fixed with

`e.preventDefault();
e.stopPropagation();`

I'll let you know if it repeats.

**Comment by aesculus** [comment]
After reviewing this more I still had some issues with the plugin. My problem was with popup panels where I registered the plugin on a click or focus handler. Worked great for the first viewing of each popup panel and then I would get this message on subsequent requests.

The solution was simple. At the close of the popup panel I deregistered (on > off) the click or focus handler. Now all is good again.
